mkdocs.yml

```diff
@@ -67,6 +67,7 @@ pages:
         - Report: reference/usage/report.md
         - Notification: reference/usage/notification.md
         - Investigating: reference/usage/investigating.md
+        - Time Constraints: reference/usage/time-constraints.md
```

Comment:
I wonder if it shouldn't be "schedule" constraints". which would be clearer. There are builtin mechanisms to timeout long operations (at least HTTP and process providers). The question is mainly around scheduling experiments.

Reply:
I can see your point, but doesn't this go further than scheduling when you consider killing experiments if they've run for too long etc?

Reply:
True. I think we could need both to be fair. Would that be okay to amend the PR to split the section into one around timing and the other one around scheduling?

Reply:
Yes indeed! Good idea, I'll do that now.